### ISSUE TITLE
[AWS] setup-network-environment hangs on the minion

### ISSUE THREAD

**Issue by roman-shuhov** [kelseyhightower/setup-network-environment#1]
I'm trying to setup kubernetes cloud in Amazon on CoreOS. I'm following this steps https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/aws-coreos.md#manually

When i try to create minions, they failed to initialize cloud config. As i see they hang at the step:

```
2015/04/08 21:10:11 Calling unit command "start" on "setup-network-environment.service"'
```

and if i try to run

```
systemctl start setup-network-environment
```

it hangs

The output of systemctl status setup-network-environment is :

```
● setup-network-environment.service - Setup Network Environment
   Loaded: loaded (/etc/systemd/system/setup-network-environment.service; static; vendor preset: disabled)
   Active: inactive (dead)
     Docs: https://github.com/kelseyhightower/setup-network-environment
```

I'm also able to execute 

/opt/bin/setup-network-environment 

but it doesn't provide any output

**Comment by roman-shuhov** [comment]
I tried to create 12 minions, and this issue occurs at 10 of them, 2 other mysterically works fine.

Issue persists even if i restart machine and try to initialize it again.

**Comment by kelseyhightower** [comment]
@roman-shuhov That command should output to /etc/network-environment

**Comment by roman-shuhov** [comment]
Ok, looks like i can fix this issue if i remove 

Requires=network-online.target and After=network-online.target

In this case all minions works fine

**Comment by roman-shuhov** [comment]
Kelsey helped me to troubleshoot this issue, it was a coreOS bug and it was fixed in  633.1 version. After i switched to this version, the issue dissapeared.

**Comment by kelseyhightower** [comment]
I'm going to close this out.
